tools/validators/instance_validator/tests/generate_universe_test.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for generate_universe.py"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from universe_generation import generate_universe
+from absl.testing import absltest
+
+class GenerateUniverseTest(absltest.TestCase):
+
+  def testCanGenerateConfig(self):
+    config = generate_universe.build_config()
```

Comment:
use SetUp for each generate_universe.build_config
https://abseil.io/docs/python/guides/testing

Reply:
Done